### ISSUE TITLE
Man page generation fails when building only partial list of drivers

### ISSUE THREAD

**Issue by Kaapeli64** [networkupstools/nut#1467]
I was writing an ebuild for Gentoo and ran into an issue whenever only some of the drivers are built, eg with:

`./configure --with-drivers=usbhid-ups`

The build will always fail:
```
asciidoc: ERROR: upsd.txt: line 185: [blockdef-filter] missing closing delimiter
a2x: ERROR: "xmllint" --nonet --noout --valid "/tmp/nut.bad/docs/man/tmp/man8.nutupsdrv.8.1443001/nutupsdrv.xml" returned non-zero exit status 4

asciidoc: ERROR: upsd.txt: line 185: [blockdef-filter] missing closing delimiter
mv: cannot stat './tmp/man8.nutupsdrv.8.1443001/nutupsdrv.8': No such file or directory
make[2]: *** [Makefile:1646: nutupsdrv.8] Error 1
make[2]: *** Waiting for unfinished jobs....
mv: cannot stat './tmp/man8.upsd.8.1443040/upsd.8': No such file or directory
make[2]: *** [Makefile:1646: upsd.8] Error 1
make[2]: *** Waiting for unfinished jobs....
mv: cannot stat './tmp/man8.upsd.8.1443085/upsd.8': No such file or directory
make[2]: *** [Makefile:1646: upsd.8] Error 1
a2x: ERROR: "xmllint" --nonet --noout --valid "/tmp/nut.bad/docs/man/tmp/man8.nutupsdrv.8.1443021/nutupsdrv.xml" returned non-zero exit status 4

mv: cannot stat './tmp/man8.nutupsdrv.8.1443021/nutupsdrv.8': No such file or directory
```
The reason is that linkman-driver-names.txt is empty.

I made a fix patch into Gentoo bug tracker https://836776.bugs.gentoo.org/attachment.cgi?id=791087 which looks sane to me. This is the first time I'm playing with autotools, so I'm not quite sure if I understood what I was doing. The results with this fix look right, the man pages contain only those drivers that were built and no more.

Regardless of the fix, the issue is real. I could try make a pull request too if you think this is done right.

**Comment by Kaapeli64** [comment]
I tried it first with just the configure.ac and the issue was still that SRC_DRIVERS_PAGES list was empty. I'm not exactly sure how that list should be populated when --with-drivers=!all, but it appears there are none in that case. This is why I added the list from DRIVER_MAN_LIST, and that appear to be working right now. But given the experience I have with autotools, I'm easily convinced otherwise otherwise. Just trying to explain my thoughts how I ended up where I got.
